plugins/synaptics-vmm9/fu-synaptics-vmm9-device.c

```diff
@@ -330,6 +330,7 @@ fu_synaptics_vmm9_device_open(FuDevice *device, GError **error)
 		g_prefix_error(error, "failed to DISABLE_RC before ENABLE_RC: ");
 		return FALSE;
 	}
+	fu_device_sleep(device, 500);
```

Comment:
is this one required? it'll slow down the daemon by 500ms on every start

Reply:
unfortunately yes, otherwise a random timing issue will be seen....

Reply:
Can you elaborate a bit? People run systemd-analyze and then open bugs when fwupd takes a "long time" to start.